brambling/templates/brambling/event/order/transfer.html

```diff
@@ -9,6 +9,14 @@
 		{{ block.super }}
 	{% endwith %}
 
+	{% if not user.is_authenticated %}
+	<div class='alert alert-danger'>
+		Sorry!
+		<b>You need to <a href="{% url 'brambling_signup' %}?email={{ order.email }}">create an account</a> to transfer an item.</b>
+		You'll be able to claim this order right away if you <a href="{% url 'brambling_signup' %}?email={{ order.email }}">create an account</a> with the same email address.
```

Comment:
I would either want this to not tell them that it needs to be the same email address, or to tell them here what email address they used on the order so they don't feel like they have to go look it up again.